docs/contributing/commit-signing.mdx

```diff
@@ -21,34 +21,38 @@ their commits.
 
 ## Setting up commit signing
 
-Github supports commit signing with GPG, SSH and S/MIME. If you're unsure what to use, we recommend
-GPG.
+Github supports commit signing with GPG, SSH and S/MIME.
 
-1.  Install GnuPG:
+:::tip
 
-  <Tabs groupId="os">
+If you're unsure what to use, we recommend you create a commit signing key using SSH. Remember to
+protect the key with a [strong passphrase or password][password-generation].
 
-    <TabItem value="mac" label="macOS">
+```bash
+ssh-keygen -f ~/.ssh/bw-signing -C "$(git config --get user.email)" -t ed25519
+```
 
-    ```bash
-    brew install gnupg
-    echo "export GPG_TTY=$(tty)" >> ~/.zshrc
-    ```
+:::
 
-    Restart your open terminal for this to take effect
+### Configuration
 
-    </TabItem>
+1. Configure git to sign using SSH.
 
-  </Tabs>
+   ```bash
+   git config --global gpg.format=ssh
+   git config --global user.signingkey=~/.ssh/bw-signing.pub
+   ```
 
-2.  Follow the [Github documentation][github-verification] to configure commit signing
+   (Remove the `--global` flag to only apply this setting to the current repository.)
```

Comment:
💭 Seems like overkill to _not_ have this global so I wouldn't even mention it.

Reply:
Addressed by 616af63a5.